SPECS/kernel/kernel.spec

```diff
@@ -259,7 +259,8 @@ echo "initrd of kernel %{uname_r} removed" >&2
 %postun
 if [ ! -e /boot/mariner.cfg ]
 then
-     if [ `ls /boot/linux-*.cfg 1> /dev/null 2>&1` ]
+     ls /boot/linux-*.cfg 1> /dev/null 2>&1
```

Comment:
It seems like all of the kernel specs utilize the same logic. I suggest creating a macro that would be equivalent to this code (e.g. `%mariner_kernel_postun`) so that any time we want to update this logic, we only have to modify a single macro file.